environment/src/main/scala/ru/primetalk/funtik/environment/geom2d/Geom2dUtils.scala

```diff
@@ -116,6 +117,48 @@ trait Geom2dUtils[V] extends Vector2dSyntax[V] {
       line(bottomRight, bottomLeft),
       line(bottomLeft, topLeft)
     ).flatten
+
+    def sideSize(axis: Axis2d): Int = axis match {
+      case Horizontal => width
+      case Vertical => height
+    }
+
+    def getLongestAxis: Option[Axis2d] = {
+      if (height > width) {
+        Some(Vertical)
+      } else if (width > height) {
+        Some(Horizontal)
+      } else {
+        None
+      }
+    }
+
+    //TODO: split onto N rectangles
```

Comment:
Do we really want to have this feature eventually?

Reply:
nope